src/main/kotlin/net/atos/zac/flowable/cmmn/AanvullendeInformatieTaskListener.kt

```diff
@@ -28,21 +28,21 @@ class AanvullendeInformatieTaskListener : TaskListener {
         LOG.fine {
             "AanvullendeInformatie task ${delegateTask.id} in state ${delegateTask.state} was ${delegateTask.eventName}"
         }
-        (delegateTask as TaskInfo).let { taskInfo ->
+        with(delegateTask as TaskInfo) {
             when (delegateTask.eventName) {
```

Comment:
is `eventName` not available in `TaskInfo`, else `delegateTask` should be removed!

Reply:
`eventName` is not part of `TaskInfo` and scopes are not fields of `DelegateTask`